src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.rickandmorty.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.List;
+
+@RestController
+@RequestMapping("/characters")
+@RequiredArgsConstructor
+public class CharacterController {
+    private final CharacterService characterService;
+
+    @GetMapping("/random-character")
+    public CharacterDto getRandomCharacter() {
+        return characterService.getRandomCharacter();
+    }
+
+    @GetMapping("/search-character/name={name}")
```

Comment:
we are already at `/character` endpoint, no need to duplicate this
```suggestion
    @GetMapping("/random")
    public CharacterDto getRandomCharacter() {
        return characterService.getRandomCharacter();
    }

    @GetMapping("/search/name={name}")
```